### ISSUE TITLE
certificate has expired or is not yet valid

### ISSUE THREAD

**Issue by MaxXx1313** [Altoros/fabric-rest#13]
```
peer0.example.com       | 2017-07-31 14:38:23.418 UTC [eventhub_producer] validateEventMessage -> DEBU 19f ValidateEventMessage starts for signed event 0xc421480de0
peer0.example.com       | 2017-07-31 14:38:23.418 UTC [eventhub_producer] Chat -> ERRO 1a0 Error handling message: event message must be properly signed by an identity from the same organization as the peer: [failed deserializing event creator: [The supplied identity is not valid, Verify() returned x509: certificate has expired or is not yet valid]]
api.example.com         | /usr/src/app/lib-fabric/peer-listener.js:44
api.example.com         |       throw e;
api.example.com         |       ^
api.example.com         | 
api.example.com         | Error: event message must be properly signed by an identity from the same organization as the peer: [failed deserializing event creator: [The supplied identity is not valid, Verify() returned x509: certificate has expired or is not yet valid]]
api.example.com         |     at ClientDuplexStream._emitStatusIfDone (/usr/src/app/node_modules/grpc/src/node/src/client.js:201:19)
api.example.com         |     at ClientDuplexStream._receiveStatus (/usr/src/app/node_modules/grpc/src/node/src/client.js:180:8)
api.example.com         |     at /usr/src/app/node_modules/grpc/src/node/src/client.js:649:14
api.example.com         | 

```

**Comment by MaxXx1313** [comment]
It might be that we have forgotten to set some propertiy/values.

**Comment by olegabu** [comment]
may to be an issue with times of x509 cert. The certs are being checked in
fabric/msp/mspimpl.go
by
https://golang.org/src/crypto/x509/verify.go line 205
I would put debug printouts in verify.go to see what time cert has for NotBefore, NotAfter and opts.CurrentTime and now variables.

**Comment by MaxXx1313** [comment]
Fixed in fabric-tools 1.1.0
